### ISSUE TITLE
`spinup-destroy.yml` file and `azure-configuration` branch missing

### ISSUE THREAD

**Issue by k-byun** [skills/continuous-delivery-azure#28]
## Describe the bug

Step 3, Activity 1, subheading "Configuring your Azure environment" instructs to review the new workflow file `spinup-destroy.yml` in `azure-configuration` branch.  However, I cannot see neither the file nor the branch being mentioned.

## To Reproduce

Follow the lab manual to the section indicated at **Describe the bug** section of this issue.

## Expected behavior

I am expeting to see both the file and the branch being mentioned at **Describe the bug** section of this issue.

## Screenshots

(no relevant screenshot; left empty intentionally)

## Device information

The issue isn't related to the device I am using.

## Additional context

1. Nothing I can think of that I can make it more clear. There seem to be some discussion that this file existed before, and some describing the reason for the absence of the file being contributed from not committing before or something.  I couldn't quite understand it. If it can be explained in simpler terms, that'd be much appreciated.
2. I tried following the lab twice from the top with fresh new sessions but I've failed on both occasions.

**Comment by vikaskumawat80** [comment]
+1

**Comment by felix-le** [comment]
+1

**Comment by zeeket** [comment]
I have the same issue. Going into the logs for the GitHub Action that (I believe) is supposed to create the `azure-configuration` I can see the following:  
  
```
Create pull request for azure-configuration into main
pull request create failed: GraphQL: Head sha can't be blank, Base sha can't be blank, No commits between main and azure-configuration, Head ref must be a branch (createPullRequest)
Error: Process completed with exit code 1.
```

**Comment by heiskr** [comment]
Thank you for opening this issue. I'm closing this issue out per https://github.com/skills/continuous-delivery-azure/pull/20#issuecomment-1445303419
